### PR TITLE
Add support for symbols in #shift_age

### PR DESCRIPTION
**Pull request by bazay**
Resolves issue: https://github.com/ruby/logger/issues/46

### CODE REVIEW

File: lib/logger/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class Logger
-  VERSION = "1.5.1"
+  VERSION = "1.5.2"
```
Comment (olleolleolle) [inline lib/logger/version.rb:4]: 👋 In order not to mix adding a feature with "managing a release", could you change the PR not to include this change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 1.5.2
```
Comment (olleolleolle) [inline CHANGELOG.md:1]:
```suggestion
# Unreleased
```

Perhaps we can collect unreleased changelog lines like this, to separate "releasing" from adding the feature?

**Review by olleolleolle — COMMENTED** [review]
Cool addition!

File: CHANGELOG.md (hunk 2 above)
Comment (bazay) [reply CHANGELOG.md]: You read my mind - just made this change as you commented 😄

File: lib/logger/version.rb (hunk 1 above)
Comment (bazay) [reply lib/logger/version.rb]: Yep totally agree

File: test/logger/test_logperiod.rb [hunk 3]
```diff
@@ -1,80 +1,67 @@
...
-    monthly_result = Logger::Period.previous_period_end(time, 'monthly')
-    month_ago = Time.parse("2018-06-30 23:59:59")
-    assert_equal(month_ago, monthly_result)
+  def assert_next_rotate_time_word(time, next_date, words)
+    assert_time_word(:next_rotate_time, time, next_date, words)
+  end
+
+  def assert_previous_period_end_word(time, previous_date, words)
+    assert_time_word(:previous_period_end, time, previous_date, words)
+  end
```
Comment (olleolleolle) [inline test/logger/test_logperiod.rb:58]: These supporting methods make the whole test very readable!

**Comment by hsbt** [comment]
@bazay Sorry to late response. I picked https://github.com/ruby/logger/commit/83502c2107c282e9e072865ab55252f2b37ec779 on master branch.

Thank you.

**Pull request closed without merging**